k8s/namespaces/jenkins/patches/ptl/cluster-00/jenkins.yaml

```
@@ -266,6 +266,17 @@ spec:
             jenkins:
               clouds:
                 - azureVM:
+                    cloudTags:
+                      - name: "application"
+                        value: "core"
+                      - name: "builtFrom"
+                        value: "hmcts/shared-services-flux"
+                      - name: "businessArea"
+                        value: "Cross-Cutting"
+                      - name: "criticality"
+                        value: "High"
+                      - name: "environment"
+                        value: "ptl"
```

https://github.com/hmcts/terraform-module-common-tags/blob/master/main.tf#L40
```suggestion
                        value: "production"
```